src/cmakeFindModules/FindFlann.cmake

```diff
@@ -2,7 +2,7 @@
 #                  Find Flann Library
 #----------------------------------------------------------
 
-FIND_PATH(FLANN_DIR flann.hpp
+FIND_PATH(FLANN_DIR flann/flann.hpp
     HINTS "${FLANN_ROOT}" "$ENV{FLANN_ROOT}" "${FLANN_INCLUDE_DIR_HINTS}"
     PATHS "$ENV{PROGRAMFILES}/flann" "$ENV{PROGRAMW6432}/flann"
```

Comment:
Not sure your fix will work for windows platforms...

Reply:
No idea either, I use Linux exclusively.